lib/mongoid/attributes.rb

```diff
@@ -335,8 +335,25 @@ def alias_attribute(name, original)
           alias #{name}_before_type_cast #{original}_before_type_cast
         RUBY
       end
+
+      # TODO: documentation
+      def unalias_attribute(name)
```

Comment:
What about adding a more generally usable  `remove_field` method that could remove both fields and aliases?

Reply:
I think that's more dangerous and has a different fundamental meaning. It's fine to remove the alias `id`. It would be risky to accidentally remove the `_id` field itself.